src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1058,12 +1089,12 @@ static SvgNode* _createSvgNode(SvgLoaderData* loader, SvgNode* parent, const cha
 static SvgNode* _createMaskNode(SvgLoaderData* loader, SvgNode* parent, const char* buf, unsigned bufLength)
 {
     loader->svgParse->node = _createNode(parent, SvgNodeType::Unknown);
+ 
     if (!loader->svgParse->node) return nullptr;
 
     loader->svgParse->node->display = false;
-#ifdef THORVG_LOG_ENABLED
-    printf("SVG: Unsupported elements used [Elements: mask]\n");
-#endif
+ 
```

Comment:
please remove empty space

Reply:
Yes I know, but now I add supprot for masking here, so log is no longer valid.

Reply:
Oh you are right. I mistook.